src/schema/order/confirm_order_mutation.js

```diff
@@ -0,0 +1,32 @@
+import { GraphQLString, GraphQLNonNull } from "graphql"
+import { OrderType } from "schema/me/order"
+import { mutationWithClientMutationId } from "graphql-relay"
+
+export default mutationWithClientMutationId({
+  name: "ConfirmOrderMutation",
+  description: "Conforms an order",
```

Comment:
ditto to previous rename comment-- should this be "ApproveOrderMutation"? In any case "Conforms" --> "Confirms" 😄